samples/hello-world/HW-ionq-qsharp.ipynb

```diff
@@ -0,0 +1,381 @@
+          }
+        }
+      },
+      "id": "92aaa651-2bfb-4989-acfb-480997e2e280"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "print(\"This workspace's targets:\")\n",
+        "for target in workspace.get_targets():\n",
+        "    print(\"-\", target.name)"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "id": "a3c691dd-5964-4699-873a-8f1b10b7650d"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### ❕ Do you see `ionq.simulator` in your list of targets? If so, you're ready to keep going.\n",
+        "\n",
+        "Don't see it? You may need to add IonQ to your workspace to run this sample. Navigate to the **Providers** page in the portal and click **+Add** to add the IonQ provider. Don't worry, there's a free credits plan available."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "id": "35eb62e5-de39-4a20-9d07-38e24bc05ba4"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## IonQ: The quantum provider\n",
+        "Azure Quantum partners with third-party companies to deliver solutions to quantum jobs. These company offerings are called *providers*. Each provider can offer multiple *targets* with different capabilities. See the table below for IonQ's targets.\n",
+        "\n",
+        "| Target name | Target ID | Number of qubits | Description |\n",
+        "| --- | --- | --- | --- |\n",
+        "| Quantum simulator | `ionq.simulator` | 29 qubits | IonQ's cloud-based idealized simulator. Free of cost. |\n",
+        "| Aria 1 | `ionq.qpu.aria-1` | 23 qubits | IonQ's Aria 1 trapped-ion quantum computer. This is real quantum hardware, not a simulation. |\n",
+        "| Quantum computer | `ionq.qpu` | 11 qubits | IonQ's trapped-ion quantum computer. This is real quantum hardware, not a simulation. |\n",
+        "\n",
+        "For this example, we will use `ionq.simulator`. To learn more about IonQ's targets, check out our [documentation](https://docs.microsoft.com/azure/quantum/provider-ionq)."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "id": "b9d29721-e717-480e-8f33-308682bacdd1"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### 2. Build the quantum program\n",
+        "\n",
+        "Let's create a simple Q# program to run.\n",
+        "\n",
+        "First, let's initialize the Q# environment and set the target profile to Base Profile. Today, Q# compilation for Azure Quantum targets only supports the Base Profile, a subset of all Q# commands."
```

Comment:
Just to consider if it would make more sense to say:
"Today, Azure Quantum targets only support the Base Profile, a subset of all Q# commands."
otherwise, it might sound like we own Q# language, but cannot compile any program.